docs/sessions/rewind.md

```diff
@@ -0,0 +1,83 @@
+# Rewind sessions for agents
+
+<div class="language-support-tag">
+  <span class="lst-supported">Supported in ADK</span><span class="lst-python">Python v1.17.0</span>
+</div>
+
+The ADK session Rewind feature allows you to revert a session to a previous
+state, enabling you to undo mistakes, explore alternative paths, or restart a
+process from a known good point. This document provides an overview of the
+feature, how to use it, and its limitations.
+
+## Rewind a session
+
+You can rewind a session to a prior invocation by using the ***rewind*** method
+on a ***Runner*** instance, specifying the user, session, and invocation, as
+shown in the following code example:
+
+```python
+# Create runner
+runner = InMemoryRunner(
+    agent=agent.root_agent,
+    app_name=APP_NAME,
+)
+
+# Create a session
+session = await runner.session_service.create_session(
+    app_name=APP_NAME, user_id=USER_ID
```

Comment:
Missing a closing parentheses here for `create_session`.

Reply:
Done.